apps/test/unit/templates/sectionsRefresh/SectionsSetUpTest.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
```

Comment:
Thanks for the tests in V1!

Reply:
Is this the kind of component we would want a storybook for? If so, would be totally fine as a followup pr.

Reply:
Oh maybe 🤔 It feels on the cusp of whether it would be useful, but I could imagine when we add the ability to create multiple sections, it might be useful. Either way, I'd rather do this as a followup -- I'm already behind on this PR 😅